test/functional/banner_setting_controller_test.rb

```diff
@@ -9,103 +9,102 @@ def setup
-                       :enable => "true", :type => "warn", :display_part => "both",
-                       :start_ymd => "2012-03-12", :use_timer => "true",
-                       :banner_description => "exp. Information about upcoming Service Interruption.",
-                       :end_hour => "23"}
+    post :plugin, id: 'redmine_banner',
+                  settings: { end_ymd: '2012-12-31', end_min: '03', start_min: '03', start_hour: '20',
+                              enable: 'true', type: 'warn', display_part: 'both',
+                              start_ymd: '2012-03-12', use_timer: 'true',
+                              banner_description: 'exp. Information about upcoming Service Interruption.',
+                              end_hour: '23' }
     assert_response :redirect
-    get :plugin, :id => "redmine_banner"
+    get :plugin, id: 'redmine_banner'
     assert_response :success
-    assert_select  'div#banner_area', false, "Banner should be off."
+    assert_select 'div.banner_area', false, 'Banner should be off.'
   end
-  
+
   def test_post_banner_settings_with_bad_format
     # set bad format
-    post :plugin, :id => "redmine_banner",
-      :settings => {:end_ymd => "2010-03-31", :end_min => "03", :start_min => "03", :start_hour => "20", 
-      :enable => "true", :type => "warn", :display_part => "both", 
-      :start_ymd => "2013-03-12", :use_timer => "true",
-      :banner_description => "exp. Information about upcoming Service Interruption.", 
-      :end_hour => "23"}
+    post :plugin, id: 'redmine_banner',
+                  settings: { end_ymd: '2010-03-31', end_min: '03', start_min: '03', start_hour: '20',
+                              enable: 'true', type: 'warn', display_part: 'both',
+                              start_ymd: '2013-03-12', use_timer: 'true',
+                              banner_description: 'exp. Information about upcoming Service Interruption.',
+                              end_hour: '23' }
     assert_response :redirect
-    assert_redirected_to :controller => 'settings', :action => 'plugin', :id => "redmine_banner"
-    assert_equal I18n.t(:error_banner_date_range), flash[:error] 
+    assert_redirected_to controller: 'settings', action: 'plugin', id: 'redmine_banner'
+    assert_equal I18n.t(:error_banner_date_range), flash[:error]
   end
 
   def test_post_banner_setting_with_non_number_format
-    post :plugin, :id => "redmine_banner",
-      :settings => {:end_ymd => "end_ymd", :end_min => "end_min", :start_min => "start_min", :start_hour => "start_hour",
-      :enable => "true", :type => "warn", :display_part => "both",
-      :start_ymd => "start_ymd", :use_timer => "true",
-      :banner_description => "exp. Information about upcoming Service Interruption.",
-      :end_hour => "end_hour"}
+    post :plugin, id: 'redmine_banner',
+                  settings: { end_ymd: 'end_ymd', end_min: 'end_min', start_min: 'start_min', start_hour: 'start_hour',
```

Comment:
__[rubocop]__
* Line is too long. [119/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)